### PR TITLE
Announce source git service shut down

### PR DESCRIPTION
**Pull request by majamassarini**
Resolve https://github.com/packit/private/issues/33

### CODE REVIEW

File: source-git/shutdown.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: Service integration for CentOS Stream shut down
+date: 2024-08-30T08:00:00+02:00
+sidebar_position: 3
+---
+# Service integration for CentOS Stream shut down
+
+The Packit team wants to let all its users know that the cluster, where the *source-git service* was running, has sadly been shut down.
+
+We can migrate the service to a new cluster but we want to be sure this big effort, for our small team, has some value.
```
Comment (lbarcziova) [inline source-git/shutdown.md:10]: :+1:

File: source-git/shutdown.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+title: Service integration for CentOS Stream shut down
+date: 2024-08-30T08:00:00+02:00
+sidebar_position: 3
+---
+# Service integration for CentOS Stream shut down
+
+The Packit team wants to let all its users know that the cluster, where the *source-git service* was running, has sadly been shut down.
```
Comment (lbarcziova) [inline source-git/shutdown.md:8]: I am wondering whether we have some docs or something that we could link here, for the *source-git service* part

File: source-git/shutdown.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+We can migrate the service to a new cluster but we want to be sure this big effort, for our small team, has some value.
+
+If you are using the *Service integration for CentOS Stream* please **reach out to the Packit team** on Matrix (#packit:fedora.im), on Slack (#team-packit)
+or email (mailto:hello@packit.dev); we can discuss the future steps together.
+
+If we will collect no feedback then the *Service integration for CentOS Stream* will be definetely shut down.
```
Comment (lbarcziova) [inline source-git/shutdown.md:15]:
```suggestion
If we collect no feedback then the *Service integration for CentOS Stream* will definitely be shut down.
```

File: source-git/shutdown.md (hunk 2 above)
Comment (majamassarini) [reply source-git/shutdown.md:8]: I can't find anything.

Comment (lbarcziova) [reply source-git/shutdown.md:8]: we could probably link [this](https://packit.dev/source-git/status#service-integration-for-centos-stream)

Comment (lbarcziova) [reply source-git/shutdown.md:8]: I know, I just meant the specific section

Comment (lbarcziova) [reply source-git/shutdown.md:8]: yes, in here add the suffix, for the last sentence have it generally

**Review by lbarcziova — APPROVED** [review]
thanks! We could share this on Mastodon as well

**Comment by majamassarini** [comment]
> thanks! We could share this on Mastodon as well

yes, I want to do it tomorrow :)

**Pull request merged**